src/models/configs.ts

```diff
@@ -56,4 +56,6 @@ export interface ReportPortalConfig extends ClientConfig, AttachmentsConfig {
   includeTestSteps?: boolean;
   includePlaywrightProjectNameToCodeReference?: boolean;
   extendTestDescriptionWithLastError?: boolean;
+  grep?: string;
+  grepInvert?: string;
```

Comment:
These parameters are not the part of the agent config, they will come from the playwright configuration, so please remove it here.